### ISSUE TITLE
PAL color loss

### ISSUE THREAD

**Issue by DirtyHairy** [stella-emu/stella#13]

**Comment by sa666666** [comment]
In implementing this, I just came across a question; when in colour-loss mode *and* debug colours mode, which should take effect?

In Stella 4.x, colour-loss took precedence, meaning that if debug colours were also activated, then we show the colour-loss equivalent of the debug colours too.  It now occurs to me that this probably isn't correct; debug colours should always be shown no matter what else is going on at that point in time.

**Comment by ale-79** [comment]
I agree. Color loss shouldn't affect debug colors, if possible.

**Comment by thrust26** [comment]
I agree too.

**Comment by sa666666** [comment]
This is now done in https://github.com/stella-emu/stella/commit/fb5daa483b4d04cd8ce15a78e485831f217a02b4.

**Comment by ale-79** [comment]
I recompiled and tried the PAL "Roc'n Rope" rom, knowing that it has an odd number of scanline and displays in B/W in most PAL TVs, but I couldn't see the effect even if I enabled it with CTRL-L.

After playing a bit with the video settings I found the cause: PAL color-loss doesn't work with TV effects enabled. Once I disabled those, everything worked fine.

I then tried Stella 4.7.3 and it's the same there. I just never noticed before, as usually PAL games that have that issue are pirate hacks which I almost never play.

I think this should be fixed, but since no one noticed before, I guess this isn't high priority and can wait future releases.
